src/plugins/telemetryTable/components/table.vue

```diff
@@ -70,13 +106,28 @@
     @import "~styles/sass-base";
     @import "~styles/table";
 
+    .js-telemetry-table__sizing {
```

Comment:
a class prefixed with .js should never be used for styling, right?

Reply:
Correct. Actually I think this whole style rule can go. Table sizing had a few iterations and this is leftover from a previous one.